src/api/todos.ts

```diff
@@ -0,0 +1,14 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const postTodo = (newTodo: Omit<Todo, 'id'>) => {
```

Comment:
Consider adding a comment above the function 'postTodo' to describe its purpose and functionality.